task_schemas.md

```diff
@@ -137,9 +139,9 @@ TBD
 	"id": "0",
 	"document_id": "24267510",
 	"question_id": "55031181e9bde69634000014",
-	"question": "Is RANKL secreted from the cells?", 
+	"question": "Is RANKL secreted from the cells?",
 	"type": "yesno",
-	"context": Osteoprotegerin (OPG) is a soluble secreted factor that acts as a decoy receptor for receptor activator of NF-\u03baB ligand (RANKL)", 
+	"context": Osteoprotegerin (OPG) is a soluble secreted factor that acts as a decoy receptor for receptor activator of NF-\u03baB ligand (RANKL)",
```

Comment:
The opening " is missing: `"context": Osteoprotegerin (OPG)` -> `"context": "Osteoprotegerin (OPG)`

Reply:
good catch